.lintstagedrc.json

```diff
@@ -1,13 +1,10 @@
 {
-  "linters": {
-    "src/**/*.{js,jsx,tsx,ts,json}": [
-      "prettier --write",
-      "eslint --fix --ext .js,.jsx,.tsx,.ts",
-      "git add"
-    ],
-    "*.md": [
-      "prettier --write",
-      "git add"
-    ]
-  }
-}
+  "extends": [
```

Comment:
are u sure this is correct? seems like config for stylelint, and not for linstaged

Reply:
please take a look at new config